WordPress/wporg-mu-plugins
# Remove `theme.json` dependency

In https://github.com/WordPress/wporg-parent-2021/issues/17, the colors in the parent theme's `theme.json` will be updated. That means some color custom properties will not be defined on sites using the parent theme. If we switch the custom properties in the header-footer CSS to those used in the new theme.json color set, then they won't be defined on News.

Rather than try to jump through hoops to load the "right" custom properties, this PR just defines them locally. While this does technically mean some duplicate colors, it also means the remote loading of global styles isn't needed anymore.

These custom properties are scoped to `.global-header` & `.global-footer` so that they don't override any theme-specific values.

**To test**

1. Spin this branch up on a sandbox
2. Visit block theme sites & classic sites, there should be no visual changes in the header or footer
3. Spin up the parent theme or a site that uses it (make sure it's the latest), and use this branch of mu-plugins
4. The header & footer should be visible
